packages/commonwealth/client/scripts/views/pages/discussions/HeaderWithFilters/HeaderWithFilters.tsx

```diff
@@ -148,8 +150,24 @@ export const HeaderWithFilters = ({
     );
   };
 
+  const handleSwitchAddress = () => {
+    console.log('click switch address');
+  };
+
+  const handleJoinCommunity = () => {
+    console.log('click handle join');
+  };
+
   return (
     <div className="HeaderWithFilters">
+      {featureFlags.sessionKeys && (
+        <SwitchAddressBanner
+          communityName="dydx"
```

Comment:
Can we note somehow that this hardcoded string needs to change?

Reply:
added comment